plasmapy/diagnostics/tests/test_langmuir.py

```diff
@@ -1,40 +1,86 @@
-"""Tests for functions that use Langmuir analysis functions."""
+# coding=utf-8
+"""Tests for Langmuir probe analysis functions."""
 
 import numpy as np
+from astropy import units as u
+import pytest
+import plasmapy.constants as const
 
-from plasmapy.diagnostics.langmuir import (swept_probe_analysis,
-                                           obtain_EEDF,
-                                           obtain_EEPF)
-
-
-class Test_swept_probe_analysis():
-    def test_no_signal(self):
-        """
-        Checks whether an error is raised when zero signal is passed to the
-        function.
-        """
-        errStr = ("Performing a swept probe analysis should return an error "
-                  "when zero signal is passed, but doesn't.")
-        assert True, errStr
-
-
-class Test_obtain_EEDF():
-    def test_no_signal(self):
-        """
-        Checks whether an error is raised when zero signal is passed to the
-        function.
-        """
-        errStr = ("Attempting to obtain the EEDF should return an error when "
-                  "zero signal is passed, but doesn't.")
-        assert True, errStr
-
-
-class Test_obtain_EEPF():
-    def test_no_signal(self):
-        """
-        Checks whether an error is raised when zero signal is passed to the
-        function.
-        """
-        errStr = ("Attempting to obtain the EEPF should return an error when "
-                  "zero signal is passed, but doesn't.")
-        assert True, errStr
+from plasmapy.diagnostics.langmuir import (Characteristic,
+                                           swept_probe_analysis)
+
+N = 30  # length of test characteristic
```

Comment:
Not sure what "test characteristic" means. This could use a more descriptive variable name.

Reply:
Actually "characteristic" is used throughout this file. It may be common terminology in Langmuir probe analysis, but describing what it means towards the beginning of the file would be useful.

Reply:
Usually one refers to the U-I curve as the Probe Characteristic. I find this more descriptive than using the terminology "curve", "trace", or "profile". I'll add a clear explanation of the usage of the word.

Reply:
That's what I thought, but didn't want to assume. :smile: 
So does length here refer to the length of the U-I array?

Reply:
Yes, it's the length of the dummy characteristics used for checking the function. Clarified in the comments.